setup.py

```diff
@@ -56,6 +56,7 @@ def find_version(*file_paths):
             'spacy',
             'nltk==3.2.5',
             'scipy',
+            'numba',
```

Comment:
@szha @leezu @eric-haibin-lin Is it okay to depend on numba?

Reply:
It's optional, but it can double the training speed.

Reply:
Given the benefit and that there is stub when numba is not available, I think it's ok.

Reply:
I agree. numba would be a great add.